src/KM_Game.pas

```diff
@@ -443,6 +443,10 @@ procedure TKMGame.GameStart(aMissionFile, aGameName: UnicodeString; aCRC: Cardin
   if fGameMode in [gmMulti, gmMultiSpectate] then
     MultiplayerRig;
 
+  // Update our ware distributions from settings
+  if fGameMode in [gmSingle, gmMulti] then
+    GameInputProcess.CmdWareDistribution(gic_WareDistributions, gGameApp.GameSettings.WareDistribution.PackToStr);
+
```

Comment:
I think it would be better to run this after we set the random seed and create the basesave. Just in case running this command uses the random number generator somehow in the future. Also the basesave is meant to be a save of the game state before we record any events through the GIP (so when watching the replay the basesave will have the default distributions and the first command replayed by the GIP will be to set the player's distributions). Basically running this command is like the player manually set all of the distributions, so we want to run it as late as possible so it's like the player did it.